manifest.php

```diff
@@ -33,7 +33,7 @@
         'generis' => '>=12.15.0',
         'tao' => '>=38.13.0',
         'taoLti' => '>=11.0.0',
-        'taoDeliveryRdf' => '>=8.3.3',
+        'taoDeliveryRdf' => '>=12.0.0',
```

Comment:
missing dependencies

Reply:
Why we increased the `taoDeliveryRdf` version? I do not remember. @bartlomiejmarszal @boajer @siwane .

Reply:
If you guys agree, we revert it.

Reply:
SHouldn't it be bump for taoLti instead ?

Reply:
taoLti has been updated to `>=11.13.0`